app/models/katello/erratum.rb

```diff
@@ -52,8 +52,9 @@ def self.applicable_to_hosts(hosts)
     end
 
     def self.applicable_to_hosts_dashboard(hosts)
+      ids = hosts.to_sql.sub(/SELECT.*FROM/, "SELECT #{::Host.connection.quote_table_name('hosts')}.#{::Host.connection.quote_column_name('id')} FROM")
```

Comment:
okay, was just trying to figure out a way to make that code a bit cleaner :/ 

Is it possible write a test that demonstrates the problem?  I'd be afraid that this code would get 'cleaned up' in the future and we wouldn't notice the breakage